README.md

```diff
@@ -0,0 +1,72 @@
+# Selenium + WebdriverIO
+## 1. lekce 
+
+Základní nastavení WebdriverIO s podporu autocomplete do VSCode. Obsahuje jeden "test" v `test/spects/01_uvod.e2e.js`
+
+### Požadavky
+* nainstalovaný git
+* nainstalované [nodejs](https://nodejs.org/en/) 
+  * Můžete zkontrolovat v terminálu příkazem `node -v`
+* webový prohlížeč [Chrome](https://www.google.com/intl/cs_CZ/chrome/)
+* [Visual Studio Code](https://code.visualstudio.com/)
+
+### Instalace 
+1. Vytvoř si složku, kde chceš mít všechny materiály - doporučuji něco ve stylu `Dokumenty/czechitas/automatizace`
+2. V terminálu jdi do vytvořené složky - pomocí příkazu `cd <dana slozka>`
+2. Stáhni si repozitář do svého počítače pomocí `git clone git@github.com:czechitas-da2020/01_automation.git`
+3. V daném adresáři pusť `npm install` 
+    * Tenhle příkaz Ti nainstaluje všechny potřebné balíčky, které pak budeme používat. 
+3. Otevři si VSCode a otevři v něm repozitář - `Dokumenty/czechitas/automatizace/01_automation`
+
+### Struktura repozitáře
+Po instalaci bys měla vidět v levém panelu několik souborů a složek. Pojďme si je vysvětlit, které jsou pro nás důležité:
+1. složka `node_modules` - zde jsou uložené všechny balíčky (programy/scripty), které jsi nainstalovala pomocí `npm install`. To je vše, co o ní potřebujeme momentálně vědět, dále s ní nebudeme pracovat.
+2. `package.json` - konfigurační soubor našeho projektu. Zde jsou uloženy všechny data, díky kterým nodejs ví, co má spouštět, jaké balíčky má stáhnout, licenci, autora atd.
+3. `wdio.conf.js` - další konfigurační soubor, nyní pro webdriverIO. Zde je uloženo nastavení, aby webdriverIO věděl, jaký prohlížeč má spustit, jaký web je jako výchozí, jaké jsou výchozí timeouty. Pokud budeme něco nastavovat a měnit, pak to bude v tomto souboru. 
```

Comment:
"timeouty" -- bude potrebovat napsat ostavec co je to timeout

Timeout - Je čas většinou nasatevený v sekundách, po který program bude čekat na nějakou operaci. 
Příklad: Timeout pro načtení webovké stránky je 10s
Pokud se do 10s webová stránka načte, nic se nestane
Pokud se do 10s webová stránka nenačte, program zahlasi chybu

Reply:
packages - balíčky? node programy?
Urcite bych tomu dal balicky

terminál? -- rikal bych tomu Prikazovy Radek / Terminal, naucime je oba dva vyrazy - ve windows se to cesky totiz jmenuje Prikazovy Radek

Nvm jestli prekladat tedn wdio.conf :-/